apps/udl-gui/udl-gui.yml

```diff
@@ -21,3 +21,4 @@ screenshots:
       caption: 'udemy-dl dashboard',
       imageLink: 'http://riazxrazor.in',
     }
+disabled: true # Dead link(s): http://riazxrazor.in & electron < v3
```

Comment:
We should probably remove this content entirely because we've gotten a DMCA for a similar app in the past: https://github.com/electron/apps/pull/1222